Brocfile.js

```diff
@@ -1,202 +1,143 @@
+var autoprefixer = require("broccoli-autoprefixer");
 var cleanCSS = require("broccoli-clean-css");
-var concatCSS = require("broccoli-concat");
+var funnel = require("broccoli-funnel");
 var env = require("broccoli-env").getEnv();
-var filterReact = require("broccoli-react");
-var jscs = require("broccoli-jscs");
-var jsHintTree = require("broccoli-jshint");
-var less = require("broccoli-less");
+var eslint = require("broccoli-lint-eslint");
+var less = require("broccoli-less-single");
 var mergeTrees = require("broccoli-merge-trees");
-var pickFiles = require("broccoli-static-compiler");
-var replace = require("broccoli-replace");
 var uglifyJavaScript = require("broccoli-uglify-js");
 var webpackify = require("broccoli-webpack");
-var _ = require("underscore");
+var packageConfig = require("./package.json");
 
-/*
- * configuration
- */
 var dirs = {
-  src: "",
-  js: "js",
-  jsDist: ".", // use . for root
-  styles: "css",
-  stylesDist: ".", // use . for root
-  img: "img",
+  src: "./src",
+  js: "./src/js",
+  dist: "",
+  styles: "./src/css",
+  img: "./src/img",
   imgDist: "img"
 };
 
-// without extensions
-var fileNames = {
+var files = {
   mainJs: "main",
   mainJsDist: "main",
-  mainStyles: "main",
-  mainStylesDist: "main"
+  mainLess: "main",
+  mainCssDist: "main",
+  index: "index.html"
 };
 
-/*
- * Task definitions
- */
 var tasks = {
-
-  jsHint: function (jsTree) {
-    // run jscs on compiled js
-    var jscsTree = jscs(jsTree, {
-      disableTestGenerator: true,
-      enabled: true,
-      logError: function (message) {
-        switch (env) {
-          case "production":
-            // TODO(mlunoe) disabled for production
-            break;
-          default:
-            // use pretty colors in test and development mode
-            console.log(chalk.red(message) + "\n");
-            break;
-        }
-      },
-      jshintrcPath: dirs.js + "/.jscsrc"
+  createJsTree: function () {
+    var jsTree = funnel(dirs.js, {
+      include: ["**/*.js", "**/*.jsx"],
+      destDir: dirs.js
     });
 
-    // run jshint on compiled js
-    var hintTree = jsHintTree(jsTree , {
-      logError: function (message) {
-        switch (env) {
-          case "production":
-            // TODO(mlunoe) disabled for production
-            break;
-          default:
-            // use pretty colors in test and development mode
-            this._errors.push(chalk.red(message) + "\n");
-            break;
-        }
-      },
-      jshintrcPath: dirs.js + "/.jshintrc"
-    });
+    return jsTree;
+  },
 
-    // hack to strip test files from jshint tree
-    hintTree = pickFiles(hintTree, {
-      srcDir: "/",
-      files: []
+  eslint: function (masterTree) {
+    return eslint(masterTree, {
+      config: ".eslintrc"
     });
-
-    return mergeTrees(
-      [jscsTree, hintTree, jsTree],
-      { overwrite: true }
-    );
   },
 
   webpack: function (masterTree) {
-    // transform merge module dependencies into one file
```

Comment:
Because else we should comment on all functions to make it consistent.
I think it's fairly self-explanatory what is webpack doing here.